examples/gatsbygram/package.json

```diff
@@ -52,6 +52,7 @@
   },
   "devDependencies": {
     "cypress": "^3.1.0",
+    "gatsby-dev-cli": "^2.0.0-rc.1",
```

Comment:
this is not needed anymore with swap to `INTEGRATION_TEST=true`?

Reply:
Correct; thanks!